enonic/tsconfig.json

```diff
@@ -6,9 +6,17 @@
     "noImplicitAny": true,
     "noImplicitReturns": true,
     "noImplicitThis": true,
-    "strictNullChecks": true
+    "strictNullChecks": true,
+    "lib": [
+      "dom",
+      "es5",
+      "scripthost",
+      "es2015.collection"
```

Comment:
What do we need from these? Especially scripthost, and es2015.collection?

Reply:
Removing